server/server_test.go

```diff
@@ -378,21 +405,23 @@ xxx row5_col1	- 	row5_col3`)
 }
 
 func runTestConcurrentUpdate(c *C) {
-	runTests(c, dsn, func(dbt *DBTest) {
-		dbt.mustExec("create table test (a int, b int)")
-		dbt.mustExec("insert test values (1, 1)")
+	// TODO: Should be runTestsOnNewDB. See #4205.
+	runTests(c, nil, func(dbt *DBTest) {
+		dbt.mustExec("drop table if exists test2")
+		dbt.mustExec("create table test2 (a int, b int)")
+		dbt.mustExec("insert test2 values (1, 1)")
 		txn1, err := dbt.db.Begin()
 		c.Assert(err, IsNil)
 
 		txn2, err := dbt.db.Begin()
 		c.Assert(err, IsNil)
 
-		_, err = txn2.Exec("update test set a = a + 1 where b = 1")
+		_, err = txn2.Exec("update test2 set a = a + 1 where b = 1")
```

Comment:
Since this test is not run in a new database, it may face conflicts when using the default table name `test`.